test/evergreen.yml

```diff
@@ -1731,135 +1744,15 @@ tasks:
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/burst_inserts_default.txt
-          test_name: burst_inserts
-
-  - name: cppsuite-bounded-cursor-perf-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config_filename: configs/bounded_cursor_perf_default.txt
-          test_name: bounded_cursor_perf
-
-  - name: cppsuite-bounded-cursor-stress-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/bounded_cursor_stress_default.txt
-          test_name: bounded_cursor_stress
-
-  - name: cppsuite-bounded-cursor-stress-reverse-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/bounded_cursor_stress_reverse_default.txt
-          test_name: bounded_cursor_stress
-
-  - name: cppsuite-bounded-cursor-prefix-stat-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/bounded_cursor_prefix_stat_default.txt
-          test_name: bounded_cursor_prefix_stat
-
-  - name: cppsuite-bounded-cursor-prefix-search-near-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/bounded_cursor_prefix_search_near_default.txt
-          test_name: bounded_cursor_prefix_search_near
-
-  - name: cppsuite-bounded-cursor-prefix-indices-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/bounded_cursor_prefix_indices_default.txt
-          test_name: bounded_cursor_prefix_indices
-
-  - name: cppsuite-reverse-split-default
-    tags: ["pull_request"]
-    depends_on:
-      - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "cppsuite test"
-        vars:
-          test_config: debug_mode=(cursor_copy=true)
-          test_config_filename: configs/reverse_split_default.txt
-          test_name: reverse_split
+          test_config: debug_mode=(background_compact,cursor_copy=true)
```

Comment:
We can remove the background_compact debug mode here. We are already testing the background debug mode with cppsuite-background-compact-default, so we don't require it here as it is the only test that enables it and we are already testing it.